src/main/resources/db/changelog/changes/08-create-shopping-carts-table.yaml

```diff
@@ -12,6 +12,8 @@ databaseChangeLog:
                   constraints:
                     primaryKey: true
                     nullable: false
+                    foreignKeyName: fk_shopping_carts_users
```

Comment:
the same remark as above

Reply:
Okay, I'll take that into account. Now there’s probably no point in adding it separately.